### PR TITLE
Provide a way to configure SMTP

### LINKED ISSUE #1: SMTP configuration
Provide a way to configure mail handler to use SMTP server, instead of sendmail.

Thanks.

### PR DESCRIPTION
**Pull request by obazoud**
Should fix #1

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -42,7 +42,9 @@
   	:to_address => node[:mail_handler][:to_address],
   	:from_address => node[:mail_handler][:from_address],
   	:send_statuses => node[:mail_handler][:send_statuses],
-  	:hostname => node[:mail_handler][:hostname]
+    :hostname => node[:mail_handler][:hostname],
+    :via => node[:mail_handler][:via],
+    :via_options => node[:mail_handler][:via_options]
```
Comment (phoolish) [inline recipes/default.rb]: I think via and via_options set to at nil in the attributes/default.rb file.

**Comment by phoolish** [comment]
Also, can you update the documentation?

**Comment by obazoud** [comment]
ready to merge ?

**Comment by phoolish** [comment]
Thanks.  I'll bump the version.

**Pull request merged**